app/http/middleware/CORSMiddleware.py

```diff
@@ -0,0 +1,34 @@
+""" CORS Middleware """
+
+from masonite.request import Request
+
+
+class CORSMiddleware:
+    """CORS Middleware
+    |--------------------------------------------------------------------------
+    | Create Dictionary of CORS headers
+    |--------------------------------------------------------------------------
+    |
+    | Appends CORS headers to HTTP response
+    |
+    | Ex. cors = {"Access-Control-Allow-Origin": "*",
+    |             "Access-Control-Allow-Headers": "Content-Type",
+    |             "Access-Control-Allow-Methods": "DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT"}
+    |
+    """
+
+    CORS = {}
```

Comment:
Shouldn't we have predefined headers here so that when we use this middleware we are up and running?

Reply:
We can use predefined, but for security questions, i told it's better allow user configure only if he needs.

I do not like the way of `Django` because it allows for default `*`, so newbies normally do not configure and all server is exposed to CORS, and lots of time, this is not needed.

On my option, it's better stay `closed` and user `open` it only if has needs

Reply:
I think also, we should be able, but don't know yet how to implement, a way to allow CORS only on specific routes, i do not know if the way `Masonite` is builded it can be done only few routes. 

But it gotta be awesome if we allow CORS only to few URL's and not all URL's.

Reply:
if we go on Django style, we can configure like this

```python
CORS = {"Access-Control-Allow-Origin": "*"}
```

Personally I don't like it. But if you think it's better do it, i can commit the update. Just tell me :)

Reply:
Yes, kind of agree with you, we need to have docs in such a way. But would wait for @josephmancuso to have an opinion here. 

Regarding you want cors to be for specific routes,
You can just do.
```
    RouteGroup([
        get('/url1', ...),
        get('/url2', ...),
        get('/url3', ...),
    ], middleware=('auth', 'jwt', 'cors')),
```

Reply:
Sure! the way you just writed is perfect.

I liked the `middleware=('auth', 'jwt', 'cors')` thing. Will be deadly simple.